pipelines/_ingredients/SingleFrame.yaml

```diff
@@ -2,7 +2,7 @@ description: Single-frame pipeline for the case in which no templates exist
 imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
     include:
-      - processCcd
+      - apPipeSingleFrame
       - initialPviCore
```

Comment:
The new `apPipeSingleFrame` includes `initialPviCore`, which suggests you probably don't need it listed here too.